ecl/eclccserver/eclccserver.cpp

```diff
@@ -254,7 +254,8 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
         {
             //Allow eclcc-xx-<n> so that multiple values can be passed through for the same named debug symbol
             const char * start = option + (*option=='-' ? 1 : 6);
-            const char * dash = strrchr(start, '-');     // position of second dash, if present
+            const char * finger = (*start=='-') ? start+1 : start; //support leading double dash
```

Comment:
I don't understand this change - can you elaborate?

Reply:
@ghalliday It's slightly confusing, but the key is that when the option is appended to the eclcc command line the stripped '-' is restored.  I was trying not to refactor the code more than I had to as this is a last minute PR.

`eclccCmd.appendf(" -%s=%s", optName.get(), value);`

Some eclcc options actually have two dashes.  "--nostdinc" for example.  So this change preserves the second dash in the option name "-nostdinc" and then both dashes end up on the command line.